services/core-api/README.md

```diff
@@ -91,23 +97,25 @@ under the `__init__.py` file.
 
 The MDS document generator service makes use of the Common Services team's [Document Generator Service](https://github.com/bcgov/common-services-team-library/tree/master/images/doc-gen-api/app).
 
-### Click commands workflow to transfer files to the object store 
-*Pre-requisite to running the transfer-files, verify-files, and reorganize-files commands: ensure that a Celery worker is running. You can use the command `ps aux` to check.
+### Click commands workflow to transfer files to the object store
+
+\*Pre-requisite to running the transfer-files, verify-files, and reorganize-files commands: ensure that a Celery worker is running. You can use the command `ps aux` to check.
 
 **I want to transfer all documents that exist on the filesystem to the object store**
- 1. Get a list of untransferred files:<br/>
+
+1.  Get a list of untransferred files:<br/>
```

Comment:
I know you didn't add them, but do we need those `<br/>` tags in here?

Reply:
strange, wondering what added this tag over here 🤔